CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ Please make sure you have:
 3. All branches have the following naming format: `iox-#[issue]-branch-name` e.g. `iox-#123-super-duper-feature`
 4. All commits have the following naming format: `iox-#[issue] commit message` e.g. `iox-#123 implemented super-duper feature`
 5. All commits have been signed with `git commit -s`
-6. You open your pull request towards the base branch `staging`
+6. You open your pull request towards the base branch `master`
 7. Link the pull request to the according Github issue and set the label accordingly
 
```

Comment:
@dkroenke We could add another section explaining our here branching strategy (master = development branch, release_x.y = stable, only hotfix, adding git tags on those branches, etc.) or do you think it's better to have this in a separate Release document?

Reply:
the branching strategy is also interesting for contributors, so i would place it in the Contributing.md